css/index.css

```diff
@@ -94,34 +94,106 @@ a:hover{
   float: left;
   margin: 0 120px 0 0;
 }
+/* Nav @media 800 */
+@media(max-width: 800px){
+.headerBox{
+  display: flex;
+  flex-direction: column;
+}
+.navBox{
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+.logoBox{
+  display: flex;
+  justify-content: center;
+  order: -1;
+  margin: 0 auto;
+}
+}
+/* Nav @media 500 */
+@media(max-width: 500px){
+  .headerBox{
+    display: flex;
```

Comment:
Once you select `display: flex` globally, it's not necessary to re-select it ion your media queries, but ONLY if you select it globally first.